_data/projects/xander.yml

```diff
@@ -0,0 +1,11 @@
+name: Xander
+desc: A Nim web application framework
+site: https://github.com/sunjohanday/xander
+tags:
+- nim
+- web
+- framework
+- library
+upforgrabs:
+  name: Help needed to keep developing the project
+  link: https://github.com/sunjohanday/xander#todo
```

Comment:
I see there's a list of things to do in the README for the project, but it'd be great to convert these to issues and provide more details, so that others can understand what needs to be done.